### ISSUE TITLE
Bulk Track Import Script to download and use.

### ISSUE THREAD

**Issue by Robby262** [LibreTime/libretime#181]
I know the team is building this functionality in, but in the meantime, this may fill a gap. A pal and I made this today and it seems to work fine. When used in a cron job, it is has similar functionality to a watched folder.

This is a pair of BASH script files which can import bulk-uploaded audio files into Libretime,
Inspired by HaJoHe's FTP Hook script.

1/ import_a_track.sh
	Imports into Libretime, a single audio file, by passing it's full absolute path.
	Usage:  import_a_track.sh /full/path/to/audiofile/myaudiofile.mp3

2/ 	import-folder-of tracks.sh
	Imports, into Libretime, all the audio files placed in /home/libretime-watched
	It calls import_a_track.sh
	The import folder is easily changed in the script.

[import_tracks.zip](https://github.com/LibreTime/libretime/files/948136/import_tracks.zip)

**Comment by hairmare** [comment]
Your description sounds like `import_a_track.sh` does exactly what [ftp-upload-hook.sh](https://github.com/LibreTime/libretime/blob/711238af18c9fe6f68754af33b4d8487570e0c15/python_apps/airtime_analyzer/tools/ftp-upload-hook.sh) should do (and calling it for multiple files is a `find` one-liner in my book). If there is an issue with the upload hook, please describe it properly so we can get it fixed.

**Comment by Robby262** [comment]
I feel that that response was rather curt, however, I actually made it for a non-techie user friend to use who had no idea how to use the old one (being a non-coder). Yes, it is just a fixed-up/rewrite, drop-in replacement version of ftp-upload-hook.sh, which now works a little better and can now be called from a loop. 
The old one would only run as a daemon and failed if you did not. The reason for that was how the api_key was recovered.

The old method returned the password as the password PLUS a line-feed and a '0 and semicolon; appended, which choked the curl statement as that was now the wrong password.

The other method also returned a complete web-page of unwanted code from /rest/method. The new one just does the rest call and returns only the json response.

Other things like simple checks and folder housework were added. I never claimed it was rocket-science, just that it works a little better and possibly of use to others as an interim bulk importer.
